### ISSUE TITLE
symbol error with gcc compile

### ISSUE THREAD

**Issue by springmeyer** [mapnik/node-mapnik#389]
Seeing:

```
node: symbol lookup error: /home/travis/build/mapnik/node-mapnik/lib/binding/node-v11-linux-x64/mapnik.node: undefined symbol: _ZN6mapnik20to_expression_stringERKNS_4util7variantIINS_10value_nullEbxdN6icu_5413UnicodeStringENS_9attributeENS_16global_attributeENS_23geometry_type_attributeENS0_17recursive_wrapperINS_10unary_nodeINS_4tags6negateEEEEENS8_INS_11binary_nodeINSA_4plusEEEEENS8_INSE_INSA_5minusEEEEENS8_INSE_INSA_4multEEEEENS8_INSE_INSA_3divEEEEENS8_INSE_INSA_3modEEEEENS8_INSE_INSA_4lessEEEEENS8_INSE_INSA_10less_equalEEEEENS8_INSE_INSA_7greaterEEEEENS8_INSE_INSA_13greater_equalEEEEENS8_INSE_INSA_8equal_toEEEEENS8_INSE_INSA_12not_equal_toEEEEENS8_INS9_INSA_11logical_notEEEEENS8_INSE_INSA_11logical_andEEEEENS8_INSE_INSA_10logical_orEEEEENS8_INS_16regex_match_nodeEEENS8_INS_18regex_replace_nodeEEENS8_INS_19unary_function_callEEENS8_INS_20binary_function_callEEEEEE
```

On travis gcc builds. This has been happening for a while but was not obvious due to the bug causing travis builds to pass even if the tests failed https://github.com/mapnik/node-mapnik/commit/6b68c4680f95a794df0f94827cab1eb424589c1f

**Comment by springmeyer** [comment]
refs https://github.com/mapbox/mapnik-vector-tile/issues/103

**Comment by springmeyer** [comment]
This is fixed now, confirmed by adding back testing #689
